kill.c

```diff
@@ -296,35 +296,64 @@ bool is_larger(const poll_loop_args_t* args, const procinfo_t* victim, procinfo_
             return false;
         }
         if (args->prefer_regex && regexec(args->prefer_regex, cur->name, (size_t)0, NULL, 0) == 0) {
-            cur->badness += BADNESS_PREFER;
+            if (args->sort_by_rss) {
+                return true;
```

Comment:
This looks wrong. If a process matches prefer_regex it will just select the first process it sees?

I suggest to instead adjust VmRSSkiB similar to what is done with BADNESS_PREFER.

Reply:
If a process matches the prefer_regex, my primary concern is to terminate it as quickly as possible, regardless of whether it utilizes the maximum virtual memory (vm).

Reply:
got it.